pnpm-lock.yaml

```diff
@@ -2,9 +2,15 @@ lockfileVersion: 5.3
 
```

Comment:
Not familiar with pnpm, not sure why my lockfile's changed so much. Changes look valid too.